scripts/templates/NOTICE.txt.tmpl

```diff
@@ -0,0 +1,29 @@
+{{- define "depInfo" -}}
+{{- range $i, $dep := . }}
+{{ "-" | line }}
+Module  : {{ $dep.Name }}
+Version : {{ $dep.Version }}
+Time    : {{ $dep.VersionTime }}
+Licence : {{ $dep.LicenceType }}
+
+{{ $dep | licenceText }}
+{{ end }}
+{{- end -}}
+
+Copyright {{ currentYear }} Elasticsearch BV
+
+This product includes software developed by The Apache Software
+Foundation (http://www.apache.org/).
+
+{{ "=" | line }}
+Third party libraries used by the Elastic APM AWS Lambda extension.
```

Comment:
Let me know if `Elastic APM AWS Lambda extension` should be something else

Reply:
Thank you for this ! I tend to use `Elastic APM Lambda Extension`, as the name of the Go package is `apm-lambda-extension`. It is also the name widely used in our first [blog post](https://www.elastic.co/blog/observability-apm-aws-lambda-serverless-functions) on the subject. However, I am aware this convention is not strictly applied everywhere, which can make things confugsing.